### DISCUSSION TITLE
Timeline Zoom Level

### DISCUSSION THREAD

**Discussion by ManifestTheData** [ivan-lednev/obsidian-day-planner#573] [Q&A]
Hello everyone :)

I would like to set the Timeline Zoom Level to below 1. My goal is to have the plugin run in my right sidebar over about half the vertical size and/but still be able to see the events/tasks of the entire day. So maybe zooming about to about half the size of level 1 would be awesome. Unfortunately, I have no experience with coding and tried with AI but couldn't get it to work.
Any chance you guys could help me with that? :pray: 

Thanks a lot in advance.

Kind regards,
MtD

**Comment by ivan-lednev** [comment]
This is impossible right now. Please consider creating a feature request.
